impl/pom.xml

```diff
@@ -18,10 +18,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-    <artifactId>attendance</artifactId>
-    <groupId>org.sakaiproject.attendance</groupId>
-    <version>23-SNAPSHOT</version>
-  </parent>
+		<artifactId>attendance</artifactId>
+		<groupId>org.sakaiproject.attendance</groupId>
+		<version>24-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
```

Comment:
was this change purposeful?

Reply:
It appeared to be common practice in Sakai to specify the relativePath to the parent module, so I added it in when updating the version to 24-SNAPSHOT. Then I figured it changed enough to warrant fixing the indentation for that section.

Reply:
sorry, i was trying to suggest that maybe these changes have broken the build!

Reply:
Is this build still failing?